js/about/adblock.js

```diff
@@ -37,6 +37,7 @@ class AdBlockItem extends ImmutableComponent {
       <SwitchControl id={this.props.resource.get('uuid')}
         rightText={this.props.resource.get('title')}
         className={`switch-${this.props.resource.get('uuid')}`}
+        customRightText={css(styles.adblockLists__adblockItem__switchControl)}
```

Comment:
This name is a bit misleading for passing in styles, Maybe call it customRightTextStyle?